src/GraphQLParser/ParserContext.Parse.cs

```diff
@@ -1418,23 +1420,23 @@ private GraphQLUnionTypeExtension ParseUnionTypeExtension(int start, List<GraphQ
         return extension;
     }
 
-    internal GraphQLValue ParseValueLiteral(bool isConstant)
+    public GraphQLValue ParseValueLiteral(bool? isConstant)
```

Comment:
I don’t understand why these were changed to nullable booleans or what null does compared to false. Here I don’t see a difference.

Reply:
I still fail to understand why it's not a `bool` instead of `bool?`.  It seems that `null` behaves identically to `false`.  Why the change?

Reply:
Yep, it works without `?`, reverted.